auth/client/src/test/resources/org/wildfly/security/auth/client/java.security

```diff
@@ -0,0 +1,14 @@
+security.provider.1=org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider file:./src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-client-default-sslcontext.xml
```

Comment:
@lvydra Thank you for pointer how to run the test! The test looks great. Just a minor. can you please include only necessary providers in this file? Since not all java versions/vendors might provide these out of the box. I think only following are needed:

security.provider.1=org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider file:./src/test/resources/org/wildfly/security/auth/client/test-wildfly-config-client-default-sslcontext.xml
security.provider.2=SUN
security.provider.3=SunJSSE

Thanks!